### PR TITLE
Fix image upload with force-bind flag

### PR DESCRIPTION
**Pull request by vasiliy-ul**
**What this PR does / why we need it**:

This PR adapts the tests to run in environments without ceph. It fixes CI failures in cgroup v2 periodic job: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirt-e2e-k8s-1.20-cgroupsv2/1464383532250959872

Additionally the PR fixes `virtctl image-upload` command with `--force-bind` flag: previously the flag wan't checked for DataVolumes causing an error if the phase was `WaitForFirstConsumer` (while the flag description states that WaitForFirstConsumer logic is supposed to be ignored) .

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by vasiliy-ul** [comment]
Hey, @brybacki , maybe you could take a look if the changes here are valid? I checked in git history that you were also working on that part.

File: pkg/virtctl/imageupload/imageupload.go [hunk 1]
```diff
@@ -445,7 +445,7 @@ func waitDvUploadScheduled(client kubecli.KubevirtClient, namespace, name string
 			return false, err
 		}
 
-		if dv.Status.Phase == cdiv1.WaitForFirstConsumer {
+		if dv.Status.Phase == cdiv1.WaitForFirstConsumer && !forceBind {
```
Comment (brybacki) [inline pkg/virtctl/imageupload/imageupload.go:448]: That is good catch, it was an actual error so it is a bug-fix, not a test fix.

Comment (vasiliy-ul) [reply pkg/virtctl/imageupload/imageupload.go:448]: I found this issue when adapted the tests to consider also Filesystem volumes. Was not sure if the flag was ignored there intentionally or not. Thanks for having a look.

File: tests/storage/imageupload.go [hunk 2]
```diff
@@ -203,15 +203,29 @@ var _ = SIGDescribe("[Serial]ImageUpload", func() {
...
+			}
+
+			if sc, exists := tests.GetCephStorageClass(); exists {
+				args = append(args,
+					"--storage-class", sc,
+					"--block-volume",
+				)
+			} else {
+				args = append(args,
+					"--access-mode", "ReadWriteOnce",
```
Comment (brybacki) [inline tests/storage/imageupload.go:223]: The only idea of this test is that when it runs on a storage with WFFC binding mode, and then CDI can force the bonding of the volume by starting worker nodes. On Immidiate storage this is basically a noop. 

Does ceph/block storage class use bindingMode = WFFC? If yes then ok (but maybe it needs a comment) if not then we could just skip the test.

Now I think, the test should "find" a storage class with a correct binding mode and only run on that storageClass.

Comment (vasiliy-ul) [reply tests/storage/imageupload.go:223]: Right, now I am thinking about why there are no errors in the CI with ceph. Probably it does not use WFFC. I tihnk I can introduce some logic here to ensure an appropriate sc with WFFC is used.

Comment (brybacki) [reply tests/storage/imageupload.go:223]: It also look that I did not check this correctly, when introducing the test

Comment (vasiliy-ul) [reply tests/storage/imageupload.go]: Updated the test so it now uses local wffc sc. Also checked that ceph indeed has volume binding mode set to `Immediate`. Now this should work in all cases whether ceph is present or not.

**Pull request merged**